packages/oui-progress/README.md

```diff
@@ -0,0 +1,111 @@
+<oui-progress type="error">
+    <oui-progress-bar value="100"></oui-progress-bar>
+</oui-progress>
+```
+
+### With labels
+
+```html:preview
+<oui-progress type="info">
+    <oui-progress-bar value="5">5%</oui-progress-bar>
+</oui-progress>
+<oui-progress type="success">
+    <oui-progress-bar value="10">10%</oui-progress-bar>
+</oui-progress>
+<oui-progress type="warning">
+    <oui-progress-bar value="60">60%</oui-progress-bar>
+</oui-progress>
+<oui-progress type="error">
+    <oui-progress-bar value="100">100%</oui-progress-bar>
+</oui-progress>
+```
+
+### Custom max value
+
+```html:preview
+<oui-progress type="success" max-value="200">
+    <oui-progress-bar value="150">150/200</oui-progress-bar>
+</oui-progress>
+```
+
+### Stacked
+
+```html:preview
+<oui-progress type="info">
+    <oui-progress-bar type="success" value="30">30%</oui-progress-bar>
+    <oui-progress-bar type="error" value="15">15%</oui-progress-bar>
+</oui-progress>
+```
+
+### Thresholds
+
+```html:preview
+<oui-progress type="info">
+    <oui-progress-threshold value="60"></oui-progress-threshold>
+    <oui-progress-bar type="error" value="80">80%</oui-progress-bar>
+</oui-progress>
+<oui-progress type="info">
+    <oui-progress-threshold value="20"></oui-progress-threshold>
+    <oui-progress-threshold value="50"></oui-progress-threshold>
+    <oui-progress-threshold value="80"></oui-progress-threshold>
+    <oui-progress-bar type="success" value="30">30%</oui-progress-bar>
+    <oui-progress-bar type="error" value="25">25%</oui-progress-bar>
+</oui-progress>
+```
+
+### Label alignment
+
+```html:preview
+<oui-progress type="info">
+    <oui-progress-bar type="success" value="30" label-align="left">Progression: 30%</oui-progress-bar>
+</oui-progress>
+```
+
+### Indeterminate
+
+```html:preview
+<oui-progress type="info" indeterminate="true"></oui-progress>
+<oui-progress type="warning" indeterminate="true"></oui-progress>
+```
+
+## API
+
+### oui-progress
+
+| Attribute               | Type            | Binding | One-time binding | Values                                 | default | Description                            |
+| ----                    | ----            | ----    | ----             | ----                                   | ----    | ----                                   |
+| type                    | String          | @       | yes              | `info`, `success`, `warning`, `error`  |         | Type of the progress component       |
+| max-value               | Number          | @?      | yes              |                                        | 100     | The max value for the progress bar    |
+| indeterminate           | Boolean         | <?      | yes              |                                        | false   | Enable/Disable indeterminate mode. When in this mode, the progress component shows an indefinite wait and the max-value would not have any effect. In this mode, the progress component should not have any other component in it |
```

Comment:
Guidelines has changed since, this is not needed

Reply:
This has been removed